packages/babel-preset-pwa-kit/package.json

```
@@ -0,0 +1,39 @@
+{
+    "name": "babel-preset-pwa-kit",
+    "version": "2.0.0-dev.5",
+    "engines": {
+      "node": "^12.0.0 || ^14.0.0",
+      "npm": "^6.14.4 || ^7.0.0 || ^8.0.0"
+    },
+    "description": "Babel preset used by PWA Kit",
+    "repository": {
+      "type": "git",
+      "url": "git+https://github.com/SalesforceCommerceCloud/pwa-kit.git"
+    },
+    "license": "SEE LICENSE IN LICENSE",
+    "author": "dev@mobify.com",
+    "main": "index.js",
+    "scripts": {
+      "lint": "npm run lint:js",
+      "lint:fix": "npm run lint:js -- --fix",
+      "lint:js": "../../node_modules/.bin/eslint \"**/*.{js,jsx}\""
```

```suggestion
      "lint:js": "eslint \"**/*.{js,jsx}\""
```
Doesn't npm automatically search parent directories? Or does it stop when it finds the package.json?

---

It doesn't resolve executables like it does resolve dependencies. This is a temporary solution to get things working. This won't work on windows. I'm looking for alternatives atm